packages/app/src/app/components/CodeEditor/VSCode/index.tsx

```diff
@@ -650,7 +652,11 @@ export class VSCode extends React.Component<Props> implements Editor {
     pushStack = false,
     model = this.editor.getActiveCodeEditor().getModel()
   ) => {
-    const results = [];
+    const results: {
+      range: any;
```

Comment:
Use `unknown` instead to have beter type safety.

```suggestion
      range: unknown;
```